Classes/OldModel/AbstractModel.php

```diff
@@ -309,7 +309,7 @@ protected function setRecordPropertyString(string $key, string $value)
      *
      * @return void
      */
-    protected function setRecordPropertyBoolean(string $key, $value)
+    protected function setRecordPropertyBoolean(string $key, bool $value)
```

Comment:
The PHPDoc type annotation actually is incorrect here. The type annotation for `$value` needs to be `mixed`, and the type declaration then should not be there.

Reply:
Could you please update the PHPDoc annotation for `$value` to `mixed`?